src/cli/init.rs

```diff
@@ -43,13 +43,19 @@ pub fn init(options: InitCommand) -> Result<(), anyhow::Error> {
         name: project_name.to_owned(),
     };
 
+    let no_git = options.no_git;
+
     match options.kind {
-        InitKind::Place => init_place(&base_path, project_params),
-        InitKind::Model => init_model(&base_path, project_params),
+        InitKind::Place => init_place(&base_path, project_params, no_git),
+        InitKind::Model => init_model(&base_path, project_params, no_git),
     }
 }
 
-fn init_place(base_path: &Path, project_params: ProjectParams) -> Result<(), anyhow::Error> {
+fn init_place(
+    base_path: &Path,
+    project_params: ProjectParams,
+    no_git: bool,
```

Comment:
Using a bool parameter is going to get messy if we put more in. This should probably be its own struct of initialization options, with just `no_git: bool` in it for now.